recipes/mafft/7.407/meta.yaml

```diff
@@ -0,0 +1,31 @@
+{% set name = "mafft" %}
+{% set version = "7.407" %}
+{% set hash = "eb8ced3391fde18576cf141117f25c27b67137eb14e5641006e6385a5cfff4f2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://mafft.cbrc.jp/alignment/software/mafft-{{ version }}-without-extensions-src.tgz
+  sha256: {{ hash }}
+  patches:
+    - osx-makefile.patch # [osx]
+    - linux-makefile.patch # [linux]
+
+build:
+  number: 0
+  skip: False
```

Comment:
This will not produce any package with skip: False. Can you remove this and see how it goes please?

Reply:
Sure!